src/StackExchange.Redis/ConnectionMultiplexer.cs

```diff
@@ -2535,7 +2538,8 @@ internal void OnManagedConnectionFailed(object sender, ConnectionFailedEventArgs
         /// <param name="log">Log to write to, if any</param>
         internal void SwitchMaster(EndPoint switchBlame, ConnectionMultiplexer connection, TextWriter log = null)
         {
-            if (log == null) log = TextWriter.Null;
+            // useless memory allocations?
+            // if (log == null) log = TextWriter.Null;
```

Comment:
Can we revert this section please, scoping the change to only one thing here?

Reply:
I've found one more mistake. Alex will send new commit next week.